### DISCUSSION TITLE
[Script request] Lubelogger

### DISCUSSION THREAD

**Discussion by Banjer** [community-scripts/ProxmoxVE#48] [Request script]
### Application Name

Lubelogger

### Website

https://lubelogger.com

### Description

Self-Hosted, Open-Source, Unconventionally-Named Vehicle Maintenance Records and Fuel Mileage Tracker

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by quantumryuu** [comment]
Assign me please! Already pushed a PR 😃
